builder/Dockerfile

```diff
@@ -12,11 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM gcr.io/google-containers/ubuntu-slim:0.8
-MAINTAINER Marcin Wielgus "mwielgus@google.com"
+FROM gcr.io/google-containers/debian-base-amd64:0.3
```

Comment:
The slim-ness of the build image probably doesn't matter as much - have you considered just using the official go images? (e.g. golang:1.8.3). This is how kubernetes releases [are built](https://github.com/kubernetes/kubernetes/blob/master/build/build-image/cross/Dockerfile).

Reply:
Done